src/pages/public/Event/EventRegister/EventRegisterSuccess.js

```diff
@@ -0,0 +1,209 @@
+
+import LinkIcon from '@material-ui/icons/Link'
+import FacebookIcon from '@material-ui/icons/Facebook'
+import Discord from '../../../../components/icons/discord.svg'
+import HouseChef from 'assets/housechef.svg'
+import SpeechBubble from 'assets/registrationpagespeech.svg'
+
+import { COLORS } from '../../../../constants/_constants/theme'
+
+const ICON_SIZE = '24px';
+const FLASH_TIME = '50';
+
+const useStyles = makeStyles(theme => ({
+  main: {
+    margin: '10px 0'
+  },
+  leftColumn: {
+    //border: '1px solid yellow'
+  },
+  rightColumn: {
+    //border: '1px solid orange',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  successMessageContainer: {
+    marginTop: '75px',
+    paddingLeft: '19px',
+    marginLeft: '13px'
+  },
+  successMessageHeading: {
+    fontWeight: 'bold',
+    fontSize: '24px'
+  },
+  whereToNextContainer: { 
+    borderLeft: `2px solid ${COLORS.BIZTECH_GREEN}`,
+    marginTop: '35px',
+    paddingLeft: '19px',
+    marginLeft: '11px'
+  },
+  whereToNextHeading: {
+    fontWeight: 'bold',
+    fontSize: '24px'
+  },
+  icon: {
+    height: ICON_SIZE,
+    width: ICON_SIZE,
+    marginLeft: '5px',
+    verticalAlign: 'bottom',
+  },
+  linkIcon: {
+    height: ICON_SIZE,
+    width: ICON_SIZE,
+    marginLeft: '5px',
+    verticalAlign: 'bottom',
+    '&:hover': {
+      transform: 'rotate(-30deg)',
+      cursor: 'pointer',
+    },
+  },
+  linkCopiedMessageContainer: {
+    paddingLeft: '19px',
+    marginLeft: '13px',
+  },
+  linkCopiedMessage: {
+    color: `${COLORS.LIGHT_YELLOW}`,
+  },
+  linkCopiedMessageHidden: {
+    color: `${COLORS.LIGHT_YELLOW}`,
+    visibility: 'hidden'
+  },
+  upcomingEventsContainer: {
+    border: `1px solid #485A78`, 
+    borderRadius: '5px',
+    padding: '19px',
+  },
+  upcomingEventsHeading: {
+    fontWeight: 'bold',
+    fontSize: '20px'
+  },
+  upcomingEventsItem: {
+    marginTop: '13px'
+  },
+  upcomingEventsEname: {
+    cursor: 'pointer',
+    color: `${COLORS.FONT_COLOR}`
+  },
+  upcomingEventsDate: {
+
+  },
+  imageContainer: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  houseChefImage: {
+    width: '30%',
+    marginLeft: '5%',
+    [theme.breakpoints.between('sm', 'md')] : {
+      width: '15%' //On small and medium screens, images need to be shrunk a bit
+    }
+  },
+  speechBubble: {
+    alignSelf: 'start',
+    width: '55%',
+    [theme.breakpoints.between('sm', 'md')] : {
+      width: '27.5%' //On small and medium screens, images need to be shrunk a bit
+    }
+  }
+}))
+
+const EventRegisterSuccess = ({
+  email, 
+  upcomingEvents, 
+  resetRegistration, 
+  history, 
+  location
+}) => {
+    const classes = useStyles();
+
+    const [displayLinkMessage, setDisplayLinkMessage] = useState(false);
+
+    let blinkingTimer = undefined;
+
+    const copyLinkToClipboard = () => {
+      navigator.clipboard.writeText(`https://app.ubcbiztech.com${location.pathname}`);
```

Comment:
maybe we can use `window.location.hostname` instead of hardcoding the base url - that way when someone is testing it on dev, it should copy https://localhost:3000 and also allows for flexibility when if we ever change our domain

Reply:
Sure. I noticed that window.location.hostname doesn't copy the port number so I used window.location.host instead.